src/xwayland/xwm/mod.rs

```diff
@@ -657,6 +660,55 @@ impl X11WM {
 
         surface.state.lock().unwrap().wl_surface = Some(wl_surface);
     }
+
+    /// Set the default cursor used by X clients
+    pub fn set_cursor(
+        &mut self,
+        pixels: &[u8],
```

Comment:
What format are the pixels supposed to be in?

Reply:
RGBA. You are right, that should be documented.